srtcore/channel.cpp

```diff
@@ -505,19 +505,7 @@ int CChannel::sendto(const sockaddr* addr, CPacket& packet) const
 
    // convert control information into network order
    // XXX USE HtoNLA!
```

Comment:
This comment belongs to this function's body now. And, would be nice if somebody finally does it.

Reply:
Moved to body.